src/Core/Domain/OrderReturnState/OrderReturnStateSettings.php

```diff
@@ -24,12 +24,14 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
-header('Expires: Mon, 26 Jul 1997 05:00:00 GMT');
-header('Last-Modified: '.gmdate('D, d M Y H:i:s').' GMT');
+declare(strict_types=1);
```

Comment:
I don't understand why git shows these files changes as if it would be the same file, instead of showing that index.php was removed separately :thinking: